spec/test_templates/json/rds_instance/rds_instance_no_echo_with_default_password.json

```diff
@@ -14,7 +14,8 @@
         "MasterUserPassword": {
           "Ref": "Password"
         },
-        "StorageEncrypted": "true"
+        "StorageEncrypted": "true",
+        "DeletionProtection": true
```

Comment:
As the new rule is failing violation, I had to update some existing test templates.